frontend/src/utils/ApiFetch.ts

```diff
@@ -1,7 +1,12 @@
 import {DEBUG} from "../pages/root.tsx";
 
+export interface ApiFetchResponse<Type>{
+    ok: boolean,
+    status_code: number,
+    content: Type
+}
 
-const ApiFetch = async (url: string, options?: RequestInit) => {
+export async function ApiFetch<Type> (url: string, options?: RequestInit) {
```

Comment:
Dit vindt ik een goede aanpassing dan moeten we geen vreemde typing dingen doen in de rest van de code.